examples/seismic/tutorials/13_making_synthetics.ipynb

```diff
@@ -0,0 +1,1504 @@
+       "                        <td id=\"T_ce21a_row4_col3\" class=\"data row4 col3\" >#9f0052</td>\n",
+       "                        <td id=\"T_ce21a_row4_col4\" class=\"data row4 col4\" >5</td>\n",
+       "                        <td id=\"T_ce21a_row4_col5\" class=\"data row4 col5\" >2.000000</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                        <th id=\"T_ce21a_level0_row5\" class=\"row_heading level0 row5\" >2</th>\n",
+       "                        <td id=\"T_ce21a_row5_col0\" class=\"data row5 col0\" >lowershale</td>\n",
+       "                        <td id=\"T_ce21a_row5_col1\" class=\"data row5 col1\" >Lower</td>\n",
+       "                        <td id=\"T_ce21a_row5_col2\" class=\"data row5 col2\" >3</td>\n",
+       "                        <td id=\"T_ce21a_row5_col3\" class=\"data row5 col3\" >#ffbe00</td>\n",
+       "                        <td id=\"T_ce21a_row5_col4\" class=\"data row5 col4\" >6</td>\n",
+       "                        <td id=\"T_ce21a_row5_col5\" class=\"data row5 col5\" >2.500000</td>\n",
+       "            </tr>\n",
+       "    </tbody></table>"
+      ],
+      "text/plain": [
+       "      surface series  order_surfaces    color  id    vp\n",
+       "3       water  Upper               1  #728f02   1  1.50\n",
+       "4   sediments  Upper               2  #443988   2  1.75\n",
+       "5  uppershale  Upper               3  #ff3f20   3  2.50\n",
+       "0         co2  Lower               1  #015482   4  1.10\n",
+       "1       sands  Lower               2  #9f0052   5  2.00\n",
+       "2  lowershale  Lower               3  #ffbe00   6  2.50"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "geo_model.add_surface_values([1.5, 1.75, 2.5, 1.1, 2., 2.5], ['vp'])\n",
+    "geo_model.surfaces"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can visualise our model, plotting data points and orientations. Then we must compute our model to interpolate the surfaces and any associated scalar fields. Then we can plot our surfaces and the associated units."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[StructuredGrid (0x7f661a8e54b0)\n",
+       "   N Cells:\t1000000\n",
+       "   N Points:\t1030301\n",
+       "   X Bounds:\t0.000e+00, 1.000e+03\n",
+       "   Y Bounds:\t0.000e+00, 1.000e+03\n",
+       "   Z Bounds:\t-1.000e+03, 0.000e+00\n",
+       "   Dimensions:\t101, 101, 101\n",
+       "   N Arrays:\t1,\n",
+       " <matplotlib.colors.ListedColormap at 0x7f66204cf110>]"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Set up plotter\n",
```

Comment:
I think you can have an embedded interactive plot within the notebook now?

Reply:
There is a `plotter_type='notebook'` option, but it raises a `NotImplementedError`